app/models/concerns/image.rb

```diff
@@ -0,0 +1,18 @@
+module Image
+  def attach_image params, call_object
```

Comment:
sao ko truyền thẳng cái params[call_object] vô đi em, bên dưới có chỗ nào dùng params khác đâu

Reply:
nó cần cái params[:action] ở check_create nữa anh ạ, nên em không thay được.

Reply:
vậy thì ở check_create truyền thẳng params[:action] đi em, chứ cần gì truyền toàn bộ params vào :v